UKTrains.indigoPlugin/Contents/Server Plugin/image_generator.py

```
@@ -120,93 +123,164 @@ def _generate_departure_image(
+	plugin_root: Path,
+	image_filename: Path,
+	text_filename: Path,
+	parameters_filename: Path,
+	departures_available: bool,
+	device,
+	logger,
+	plugin_prefs=None
+) -> bool:
+	"""Generate departure board image(s) based on plugin configuration.
+
+	Can generate classic (720×400 landscape) and/or modern (414×variable portrait)
```

Ruff (0.14.14)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)